packages/manager-components/src/components/error/error.spec.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-return-await */
+import React from 'react';
+import { render, waitFor } from '../../utils/test.provider';
+import { ErrorBanner } from './error.component';
+import { ErrorObject, ErrorBannerProps } from './error.types';
+import { defaultProps } from './error.stories';
+
+const setupSpecTest = async (
+  customProps?: Partial<ErrorBannerProps>,
+  error?: ErrorObject,
+) =>
+  await waitFor(() =>
+    render(
+      <ErrorBanner error={error ?? defaultProps.error} {...customProps} />,
+    ),
+  );
+
+describe('specs:msc-error', () => {
```

Comment:
It is not `msc-error` anymore. Correct the component name.

Reply:
👍